.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.16.0
+erlang 26.0
```

Comment:
I don't think we should include a tools version here as Ethers is not deployable on it's own and should be working with any version higher then the minimum required specified in `mix.exs`.

Reply:
I think we can merge after this 🚀